### DISCUSSION TITLE
How to flow text between two containers?

### DISCUSSION THREAD

**Discussion by iacore** [typst/typst#3229] [Q&A] [answered]
As of writing, I only know how to flow text across column and page. Is there a way that I can flow text between any two rectangular regions?

reference in CSS: flow-into https://caniuse.com/css-regions

**Comment by Enivex** [comment, marked as answer]
No, this is not possible (yet)

**Reply by iacore** [reply]
how would i support it?

**Reply by Enivex** [reply]
> how would i support it?

What do you mean? Contributing this feature to typst?

**Reply by iacore** [reply]
yes

**Reply by Enivex** [reply]
> yes

First off I'd start by joining the Discord
